avocado_vt/options.py

```diff
@@ -34,6 +34,9 @@
 from virttest.standalone_test import SUPPORTED_TEST_TYPES
 
 
+LOG = logging.getLogger('avocado.vt.options')
```

Comment:
It would be better if we use small caps here - it is both more standard and we might confuse LOG with one of the avocado outputs like LOG_UI and LOG_JOB. Unless you maybe mean "LOG_VT_OPTS" or something but a simple `log = logging.getLogger(...)` is better IMO.

Reply:
Okay, that makes sense for me, will update later.

Reply:
After considering it again, since it's a global variable, I will fall back to use big caps, but I will use another name.